packages/loaders/src/styled/StyledSVG.ts

```diff
@@ -14,6 +14,8 @@ interface IStyledSVGProps {
   fontSize?: string | number;
   width: number | string;
   height: number | string;
+  containerWidth?: string;
+  containerHeight?: string;
```

Comment:
Out of curiosity, is there any specific reason for using `containerWidth` instead of `width`?

Reply:
There's a viewport surrounding the SVG dimensions that serves as a frame. 

[Width and height power Viewbox](https://github.com/zendeskgarden/react-components/pull/1233/files#diff-e79c04ddf2586ab7abb3c833519c98a1224b0e5b12edb2582a65ede9532f10ddR27)

In my opinion if we didn't mind breaking changes I would refactor `width` to `viewBoxWidth` and `height` to `viewBoxHeight`. Then `containerWidth|Height` would be just `width` and `height`.

Reply:
These are `styled` component adaptations. We use whatever utilitarian naming serves the purpose here (as long as the props don't leak into unwanted HTML attributes). These components are not exported nor subject to a breaking change discussion.

Reply:
Im going to leave this alone as I tried prop name refactor quickly however it caused more problems than the scope of this effort.